src/components/axes/timeAxis.ts

```diff
@@ -141,6 +141,8 @@ export module Axis {
     private _tierMarkContainers: D3.Selection[];
     private _tierBaselines: D3.Selection[];
     private _tierHeights: number[];
+    private _currentTimeAxisConfigurations: TimeAxisConfiguration[];
```

Comment:
Should I just leave it as it was before? I thought that `_possibleTimeAxisConfigurations` is an inappropriate name